### PR TITLE
Adds logger based on existing configuration

### PR DESCRIPTION
**Pull request by arothuis**
[Ticket 111131930](https://www.pivotaltracker.com/story/show/111131930)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,6 +10,11 @@
         "issues": "https://github.com/OpenConext/OpenConext-engineblock/issues",
         "source": "https://github.com/OpenConext/OpenConext-engineblock"
     },
+    "config": {
```
Comment (DRvanR) [inline composer.json]: Please remove this section for now

File: src/OpenConext/EngineBlock/CompatibilityBundle/Resources/config/services.yml [hunk 2]
```diff
@@ -37,6 +37,36 @@ services:
...
+        arguments:
+            - @eb.compat.log.line_formatter
+
+    eb.compat.log.session_id_processor:
+        class: EngineBlock_Log_Monolog_Processor_SessionIdProcessor
+        tags:
+            - { name: monolog.processor }
+
+    eb.compat.log.request_id_processor:
+        class: EngineBlock_Log_Monolog_Processor_RequestIdProcessor
```
Comment (DRvanR) [inline src/OpenConext/EngineBlock/CompatibilityBundle/Resources/config/services.yml]: Would like to see this and the other classes `EngineBlock_*` ported (moved and renamed, incl. tests etc) to the authenticationbundle.

Comment (DRvanR) [inline src/OpenConext/EngineBlock/CompatibilityBundle/Resources/config/services.yml]: Ignore this for now, added an issue to do this: https://www.pivotaltracker.com/story/show/112492653

**Comment by arothuis** [comment]
@DRvanR

**Comment by DRvanR** [comment]
:+1:

**Pull request merged**